CMakeLists.txt

```diff
@@ -63,7 +63,10 @@ if (UNIX)
 	# force 32bit build for now and set necessary flags to compile code as is
 	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -m32 -std=gnu99 -fno-omit-frame-pointer")
 	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -m32 -std=gnu++11 -fno-omit-frame-pointer")
-	set(CMAKE_SHARED_LINKER_FLAGS "-m32")
+	set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -m32")
```

Comment:
I haven't seen any official (or unofficial, for that matter) word saying it should be done like you suggested. While I understand the difference between string vs list approach, I fail to see any advantage of lists for cmake here. If you can name such, I can consider your suggestion, but otherwise I'd rather stay with the current approach as it is much more common.